tests/unit/routeSteps.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { expect } from "chai";
+import { describe, it } from "mocha";
+import { ChaiRequest } from "../config";
+import { MongoClient } from 'mongodb';
+require('../config'); // recommended way of loading root hooks
+
+const url = '/routeSteps';
+
+// set up dbUrl
+let dbUrl = process.env.MONGO_DB_URI;
+dbUrl = dbUrl!.substring(0, dbUrl!.lastIndexOf('--TEST') )
+
+const client = new MongoClient( dbUrl, { useUnifiedTopology: true } );
+
+async function clientConnect() {
```

Comment:
doesnt' need to be a function
you can just write
```
await CLIENT.connect().catch(console.error);
```

in line 29

Reply:
addressed as suggested